### PR TITLE
Updated core and react README for atlantic-2

### PR DESCRIPTION
**Pull request by carsonfalcon**

### CODE REVIEW

File: packages/core/README.md [hunk 1]
```diff
@@ -54,6 +54,20 @@ const client = await getSigningClient({
 });
 ```
 
+
+#### Cosm Wasm and Signing Cosm Wasm Client
+
+The `SeiSigningCosmWasmClient` and `SeiCosmWasmClient` can be used to interact with CosmWasm smart contracts. Typically, you need the `signingCosmWasmClient` to execute contract messages and the `cosmWasmClient` to query contract state.
+
+```typescript
+import { SeiCosmWasmClient, SeiSigningCosmWasmClient, connect } from '@sei-js/core';
```
Comment (besated) [inline packages/core/README.md:63]: there's also helpers `getCosmWasmClient` and `getSigningCosmWasmClient`

**Review by besated — APPROVED** [review]
left a couple comments, but looks good!

File: packages/core/README.md (hunk 1 above)
Comment (carsonfalcon) [reply packages/core/README.md:63]: Where are these? I see those helper methods in the proto library, but that proto package didn't have the Tendermint 35 fixes I believe.

Comment (carsonfalcon) [reply packages/core/README.md]: Okay, cool I fixed this.

**Pull request merged**